packages/ansible.rb

```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Ansible < Package
+  description 'Ansible is a radically simple IT automation engine that automates cloud provisioning, configuration management, application deployment, intra-service orchestration, and many other IT needs.'
+  homepage 'https://www.ansible.com/'
+  version '2.4.3'
+  source_url 'http://releases.ansible.com/ansible/ansible-2.4.3.0.tar.gz'
```

Comment:
Can we use https://releases.ansible.com/ansible/ansible-2.4.3.0.tar.gz? (https instead of http)

Reply:
Of course we can ! Let me change that